mitosheet/src/components/taskpanes/Graph/GraphStyleTab.tsx

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) Saga Inc.
+
+import React from 'react';
+import { GraphParams } from '../../../types';
+import Input from '../../elements/Input';
+import Toggle from '../../elements/Toggle';
+import Col from '../../spacing/Col';
+import Row from '../../spacing/Row';
+
+/* 
+    Contains all of the options for styling graphs,
+    like setting the title and axis labels
+*/
+function GraphStyleTab(props: {
+    graphParams: GraphParams
+    setGraphParams: React.Dispatch<React.SetStateAction<GraphParams>>;
+    setGraphUpdatedNumber: React.Dispatch<React.SetStateAction<number>>;
+}): JSX.Element {
+
+    const graphStylingParams = props.graphParams.graphStyling
+
+    return ( 
+        <div className='graph-sidebar-toolbar-content'>   
+            <div>
+                <div className='text-header-2'>
+                    Title
+                </div>
+                <Row justify='space-between' align='center'>
+                    <Col>
+                        <p>
+                            Title
+                        </p>
+                    </Col>
+                    <Input 
+                        value={graphStylingParams.title.title || ''}
+                        placeholder="Default Graph Title"
+                        onChange={(e) => {
+                            props.setGraphParams(prevGraphParams => {
+                                const graphParamsCopy: GraphParams = JSON.parse(JSON.stringify(prevGraphParams)); 
+                                return {
+                                    ...graphParamsCopy,
+                                    graphStyling: {
+                                        ...graphParamsCopy.graphStyling,
+                                        title: {
+                                            ...graphParamsCopy.graphStyling.title,
+                                            title: e.target.value !== '' ? e.target.value : undefined
```

Comment:
You cannot use `e.target.value` inside of a callback. We've seen this as a sheet crashing bug a couple of times - generally, you have to a access events _immediately_ in react. This is hard to remember!

At the start of the onChange function, just add `const title = e.target.value` - and then use it below!